lib/sycamore/sycamore/transforms/partition.py

```diff
@@ -405,7 +417,8 @@ def __init__(
         threshold: Optional[Union[float, Literal["auto"]]] = None,
         use_ocr: bool = False,
         ocr_images: bool = False,
-        ocr_tables: bool = False,
+        ocr_model: str = "easyocr",
+        per_element_ocr: bool = False,
```

Comment:
I know we had some discussions about what the default should be for this. I thought you had said that maybe we would continue to do per_element_ocr by default. Is that still the case?

Reply:
That's true, good catch. I'll update it.